### DISCUSSION TITLE
how to transfer btc out of an ord wallet?

### DISCUSSION THREAD

**Discussion by punk3700** [ordinals/ord#1622] [General]
i sent a bit more than i actually spent.. to my ord wallet.

what is the cmd line to transfer the fund out?

**Comment by raphjaph** [comment]
`ord wallet send <ADDRESS> <AMOUNT>`

For the <AMOUNT> you can do for example 0.1btc or 2000sat

**Reply by happyluckbox** [reply]
Hi, what if we already have inscriptions on the ord wallet? What's the best way to send only BTC out of the ord wallet, without touching the inscriptions?

**Reply by raphjaph** [reply]
This is the way. The <AMOUNT> field is actually a <OUTGOING> field. It'll recognize if you are sending an inscription or if you are sending pure btc. The `ord wallet` will make sure you do not send an inscription if you do not explicitly tell it to.
